backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/window/model/Document.java

```diff
@@ -2164,7 +2164,7 @@ public Document build()
 
 			final DocumentEntityDescriptor entityDescriptor = getEntityDescriptor();
 			final ITabCallout documentCallout = entityDescriptor.createAndInitializeDocumentCallout(document.asCalloutRecord());
-			document.documentCallout = ExceptionHandledTabCallout.wrapIfNeeded(documentCallout);
```

Comment:
what is the background of this change?

Reply:
The user was still saved because the exception wasn't handled properly